### DISCUSSION TITLE
Can the Google rendered consent form be translated to another language?

### DISCUSSION THREAD

**Discussion by jeremyhalin** [invertase/react-native-firebase#5776] [Q&A] [answered]
I would like to display it in user's language.

**Comment by mikehardy** [comment, marked as answer]
I'm not sure what you mean by "consent form" as there's no indication of what exactly triggers it, or what versions you are on or anything. However just guessing that you mean admob as there is no other item that needs consent I think, then likely this won't receive attention here as AdMob was completely removed from the firebase ecosystem - #5329

**Reply by jeremyhalin** [reply]
Sorry for the largely unclear question. Yes I was talking about the AdMob Google consent form.
At the moment, the answer is clearly no. One solution would be to implement my own consent form, as explained in the documentation. Maybe Google will provide this consent form in multiple language in the future, and it may be implemented in the new repo.
Thanks for your reply. I will closely follow the development of the [AdMob only repo](https://github.com/invertase/react-native-google-ads/).
